internal/pkg/cli/dialog/use_template.go

```diff
@@ -64,24 +64,31 @@ func (p *Dialogs) AskUseTemplateOptions(inputs input.Inputs) (results map[string
 			if i.Default != nil {
 				selectPrompt.Default = i.Default.(string)
 			}
-			value, _ := p.Select(selectPrompt)
-			ctx = context.WithValue(ctx, contextKey(i.Id), value)
-			results[i.Id] = value
+			selectedOptionName, _ := p.Select(selectPrompt)
+			selectedValue := input.MapOptionsByName(i.Options)[selectedOptionName]
```

Comment:
Přemýšlel jsem jestli je tady potřeba kontrolovat že `selectedOptionName` opravdu je v tom poli, ale zřejmě se tam nemá jak dostat nevalidní hodnota a koukám do `cli/dialog/configs` že se to tam taky neřeší, tak snad dobrý. 🙂

Reply:
Rozmyslam, ze by bolo asi dobre vzdy ten `err` vratit, a formular ukoncit.
Viem ze sa niekde inde v dialogoch tieto chyby ignoruju, ... no tam sa spoliehame na default value, ktora je validna, ... alebo sa to cele este nakoniec validuje, napr:
![image](https://user-images.githubusercontent.com/19371734/148787675-ee93e1ef-59c6-4785-91dd-7ea2234dd3c6.png)

Reply:
Aha, tak jsem to asi ještě úplně nepochopil. 😅 Jaký je rozdíl mezi `Select` a `SelectIndex`? Myslím hlavně v použití.

Reply:
Takhle nějak? https://github.com/keboola/keboola-as-code/pull/419/commits/f4aa9f4588b91dabbebe6100ebeaf70e9f4d5438

Reply:
Rozdiel je v tom ci ti to vrati vybranu hodnotu, alebo cislo od `0`, ktora polozka v poradi bola vybrana.
Dopln prosim komentar k metodam aby to bolo jasnejsie.

`Select` pouzivas pri jednoduchych veciach, napr. ci sa ma vytvorit `branch/config/row` ... 
Ak ti to pride prehladnejsie, tak `select` mozeme zrusit, a zostane iba `selectIndex -> select`.

Reply:
Aha, ok. Rozdíl ve fungování jsem pobral, jen mi nebylo jasný proč jsou tu na to ty dvě různý (v čem se liší jejich use casy). Ale už to asi chápu. 🙂 Ještě teda přihodím to odchytávání chyb jak jsi tu psal někde vedle.

Reply:
> Rozmyslam, ze by bolo asi dobre vzdy ten `err` vratit, a formular ukoncit. Viem ze sa niekde inde v dialogoch tieto chyby ignoruju, ... no tam sa spoliehame na default value, ktora je validna, ... alebo sa to cele este nakoniec validuje, napr: ![image](https://user-images.githubusercontent.com/19371734/148787675-ee93e1ef-59c6-4785-91dd-7ea2234dd3c6.png)

Je to tam: https://github.com/keboola/keboola-as-code/pull/419/commits/ecac4a9dc5ed36ad3d866e9ce1b4fad546f45f6b